src/sonic-yang-models/yang-models/sonic-device_neighbor_metadata.yang

```diff
@@ -0,0 +1,100 @@
+module sonic-device_neighbor_metadata {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-device_neighbor_metadata";
```

Comment:
We should use sonic-net to replace Azure

Reply:
Good catch, this is a cherry-pick PR, let's fix the issue in master then cherry-pck. Thanks.